applications/facings-nft-collection-manager.md

```diff
@@ -0,0 +1,233 @@
+- **Team Name:** FACINGS, Inc.
+- **EOS Payment Address:** facings
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 3
+- **Pomelo Grant(s):** https://pomelo.io/grants/eospyo
+- **Project is Open-Source:** Yes
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** https://github.com/FACINGS/nft-collections-manager
+
+
+## Contact
+
+- **Contact Name:** Rob Konsdorf
+- **Contact Email:** rob@facings.io
+- **Website:** https://facings.io
+
+
+## Project Overview
+
+Our goal with the NFT Collection Manager is to create a publishing platform that leverages EOS’s robustness and attracts gaming studios to a more simplistic and user friendly launchpad. We believe the integration of the Collection Manager with additional features down the road will be a very compelling choice, supporting EOS in its endeavor to become a premier gaming blockchain, while maintaining excitement for other trends and collections. 
+
+### Overview
+
+- **Name:** NFT Collection Manager
+- **Brief Description:** An interface to the AtomicAssets standard which makes it easy to manage your own Atomic NFT Collections, with draft mode and collaboration features to ensure high quality releases and gamification integration.
+- **Relationship to EOSIO:** FACINGS has been launching AtomicAssets NFTs  since its inception in early 2021. Its parent company and initial investor, Detroit Ledger Technologies (formerly EOS DETROIT), has been in the EOSIO ecosystem since its inception as a pioneer block producer on EOS and other EOSIO blockchains.
+- **Reason for Interest:** Our team and experience combined with the need for an open-source creator that is friendlier to use makes for a perfect fit. The lack of development on EOS for NFTs and gamification resources makes it a necessity going forward.
+
+
+### Project Details
+
+**Mock-ups/designs of any UI components**
+
+These designs are from past prototypes, to give an example, but does not represent the final product: 
+https://drive.google.com/drive/folders/1iN2bQJhB_w_A_fKXC8-iYr3Re2-rbqSt
+
+**Data models of the core functionality**
+
+The data model will be largely based on the AtomicAssets [schema](https://github.com/pinknetworkx/atomicassets-contract/wiki/Tables).
+
+**API specifications of the core functionality**
+
+We seek to implement a large portion of the [AtomicAssets](https://github.com/pinknetworkx/atomicassets-contract/wiki/Actions) allowed actions in the NFT Collection Manager UI bringing together the power of existing actions into the EOS framework for implementation with complex collections and gamification.
+
+
+**An overview of the technology stack to be used**
+
+The NFT Collection Manager will use React/JS. Our goal is to minimize dependencies. Therefore we intend for all of the data to be stored on-chain. In the future, IPFS will be used to store all NFT images and a testnet is used to stage draft collections.
```

Comment:
> IPFS will be used to store NFT images

Would it be possible to include [Pinata](https://www.pinata.cloud/) API support when uploading images to IPFS.

For the public use platform, would there be any rate limit mechanism to prevent API abuse?

Reply:
I think both suggestions make sense, I will make sure we let people bring their own Pinata API keys. As far as the rate limit goes, we will need one as a publicly available resource.